integration_tests/sdk/setup_integration.py

```diff
@@ -195,6 +202,8 @@ def setup_data_integrations(filter_to: Optional[str] = None) -> None:
             _setup_snowflake_data(client, integration)
         elif integration.type() == ServiceType.S3:
             _setup_s3_data(client, integration)
+        elif integration.type() == ServiceType.POSTGRES:
```

Comment:
You can change this to `elif isinstance(integration, RelationalDBIntegration):` to catch all relational databases.

Reply:
good catch, seeing something different in main though:
```
if integration.type() in [
    ServiceType.BIGQUERY,
    ServiceType.REDSHIFT,
    ...
]```

Reply:
I made this change in my PR. I just merged it with [main](https://github.com/aqueducthq/aqueduct/blob/da6bad634708190e4ec4931aa0e5d4d47e5b43ff/integration_tests/sdk/setup_integration.py#L185) so I think if you rebase to main it will show up now. :^)

Reply:
great :)